bundle/compliance/cis_gcp/test_data.rego

```diff
@@ -12,6 +12,15 @@ generate_iam_policy(members, role) = {
 	"subType": "gcp-iam-service-account",
 }
 
+generate_gcp_asset(type, subtype, resource, iam_policy) = {
```

Comment:
IMO using this alone (without more generator functions) is preferable because it keeps the test isolated, and all the relevant and rule-specific code is in one place. also removes the need for more functions